app/controllers/application_controller.rb

```diff
@@ -4,6 +4,9 @@
 
 class ApplicationController < ActionController::Base
   include Pundit
+  after_action :verify_authorized, except: :index
```

Comment:
✨

Reply:
rubocop wants to tell us that  `index` is not explicitly defined on the class. 
but i'm okay with this

Reply:
What happens if it's used in a controller that doesn't define `:index`?